apps/accounts/utils/general.py

```diff
@@ -498,3 +503,34 @@ def resolve_encrypted_uid(encrypted_uid):
     except UserModel.DoesNotExist:
         user = None
     return user
+
+
+def parse_authorization_header(
```

Comment:
This is better in https://github.com/hacktoolkit/django-htk/blob/master/utils/request.py